src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -24,107 +24,107 @@
           text: >
             az vmss create -n my_vmss_name -g myrg --dns-name-for-public-ip myGloballyUniqueDnsName
             --load-balancer myLoadBalancer --vnet-name myVNET --subnet mySubnet --image UbuntuLTS
             --generate-ssh-keys
-        - name: Scaleset created from custom Linux image using existing ssh public key of ~/.ssh/id_rsa.pub
+        - name: Create a Linux VM scale set from a custom image using an existing ssh public key of ~/.ssh/id_rsa.pub.
           text: >
             az vmss create -n my_vmss_name -g myrg --image my_linuximage_in_myrg
-        - name: Scaleset created with a cloud-init script (https://docs.microsoft.com/en-us/azure/virtual-machines/virtual-machines-linux-using-cloud-init)
+        - name: Create a Linux VM scale set with a cloud-init script (https://docs.microsoft.com/azure/virtual-machines/virtual-machines-linux-using-cloud-init).
           text: >
             az vmss create -g group_name -n my-vmss-name --image debian --custom_data ./my-cloud-init-script.yml
 """.format(image_long_summary)
 
 helps['vm availability-set create'] = """
     type: command
-    short-summary: Create an availability set
-    long-summary: For more info, see https://blogs.technet.microsoft.com/yungchou/2013/05/14/window-azure-fault-domain-and-upgrade-domain-explained-explained-reprised/
+    short-summary: Create an Azure Availability Set.
+    long-summary: For more information, see https://docs.microsoft.com/azure/virtual-machines/virtual-machines-linux-manage-availability.
     examples:
-        - name: Create Availability Set
+        - name: Create an availability set.
           text: az vm availability-set create -n av_set_name -g group_name
 """
 
 helps['vm availability-set update'] = """
     type: command
-    short-summary: Update an availability set
+    short-summary: Update an Azure Availability Set.
     examples:
-        - name: Update Availability Set
+        - name: Update an availability set.
           text: az vm availability-set update -n av_set_name -g group_name
-        - name: Update Availability Set Tag
+        - name: Update an availability set tag.
           text: az vm availability-set update -n av_set_name -g group_name --set tags.foo=value
-        - name: Remove Availability Set Tag
+        - name: Remove an availability set tag.
           text: az vm availability-set update -n av_set_name -g group_name --remove tags.foo
 """
 
 helps['vm availability-set convert'] = """
     type: command
-    short-summary: convert an availability set to contain VMs with managed disks
+    short-summary: Convert an Azure Availability Set to contain VMs with managed disks.
     examples:
-        - name: Convert AV Set with unmanaged disks to managed by resource group and name
+        - name: Use resource group and name to convert an availability set with unmanaged disks to use managed disks.
```

Comment:
Seems unnecessary to point out the parameters to use here. I think it could just say "Convert an availability set with..."